goth_tests/e2e/vm/test_e2e_vm.py

```diff
@@ -0,0 +1,92 @@
+"""End to end tests for requesting VM tasks using goth REST API client."""
+
+import json
+import logging
+import os
+from pathlib import Path
+from typing import List, Optional
+
+import pytest
+
+from goth.address import (
+    PROXY_HOST,
+    YAGNA_REST_URL,
+)
+from goth.configuration import load_yaml, Override
+from goth.node import node_environment
+from goth.runner import Runner
+from goth.runner.container.payment import PaymentIdPool
+from goth.runner.container.yagna import YagnaContainerConfig
+from goth.runner.probe import ProviderProbe, RequestorProbe
+
+from goth_tests.helpers.negotiation import DemandBuilder, negotiate_agreements
+from goth_tests.helpers.activity import vm_exe_script
+
+logger = logging.getLogger("goth.test.e2e_vm")
+
+
+@pytest.mark.asyncio
+async def test_e2e_vm(
+    common_assets: Path,
+    config_overrides: Optional[List[Override]],
+    log_dir: Path,
+):
+    """Test successful flow requesting a Blender task with goth REST API client."""
+
+    goth_config = load_yaml(common_assets / "goth-config.yml", config_overrides)
+    task_package = (
+        "hash:sha3:9a3b5d67b0b27746283cb5f287c13eab1beaa12d92a9f536b747c7ae:"
+        "http://3.249.139.167:8000/local-image-c76719083b.gvmi"
+    )
```

Comment:
I think a couple of test cases would benefit if `test_package` were provided by a common fixture.

Reply:
I moved them to `helpers/activity.py` instead, since that's where we already take our exe scripts from. This was updated in 09dfb867e168fadc03910d24e1e9230711c5d5fe